main.go

```diff
@@ -224,3 +249,32 @@ func validateConfig(imgConfig *cisoperatorapiv1.ScanImageConfig) error {
 	}
 	return nil
 }
+
+func validateCustomScanHostPaths(hostPaths []string) error {
+	protectedDirs := []string{"/bin", "/boot", "/dev", "/etc", "/lib", "/lib64", "/proc", "/root", "/run", "/sbin", "/selinux", "/sys", "/tmp", "/usr", "/var"}
+
+	hostPathSet := make(map[string]bool)
+
+	for _, path := range hostPaths {
+		if !filepath.IsAbs(path) {
+			return fmt.Errorf("path must be absolute: %s", path)
+		}
+
+		if path == "/" {
+			return fmt.Errorf("root path '/' is not allowed")
+		}
+
+		for _, protected := range protectedDirs {
+			if path == protected || strings.HasPrefix(path, protected+"/") {
```

Comment:
This will probably not work very well if executed in Windows. Perhaps `string(filepath.Separator)` could be used instead. Either way, it would be good to have unit tests showing all the inputs and outputs we have considered.

```suggestion
			if path == protected || strings.HasPrefix(path, protected+"/") {
```